tfschema/tfschema.go

```diff
@@ -63,11 +65,23 @@ func detectProviderName(name string) (string, error) {
 }
 
 func getClient(providerName string, dir string) tfschema.Client {
+	logger := hclog.New(&hclog.LoggerOptions{
+		Name:   "plugin",
+		Level:  hclog.Trace,
+		Output: hclog.DefaultOutput,
+		// this annoyance - both tfschema and go-plugin open output
+		// directly to os.Stderr, bypassing our log level filter.
+		// weird to need to bypass the issue by assigning the default
+		// output ¯\_(ツ)_/¯
```

Comment:
Should we consider opening a PR for `tfschema` in the meantime/instead?

Reply:
PR was opened and accepted - it allows overriding the default.
it doesn't make sense to PR this "not using default" issue: the code in tfschema tries to open RPC connection first, using go-plugin, and fallback to "net" method otherwise. in the first case, the code in go-plugin (hashicorp) is responsible for creating the logger. in the second case, the code to open the logger was copied as is to tfschema. so to get this PR fully:
1. need to PR go-plugin (if i have to guess, not very likely to happen anytime soon)
2. PR tfschema
so i took the pragmatic approach - which is to override the logger myself. just wanted to point out why the code is specifically using "DefaultLogger", since coming back here later will raise an eyebrow